code/_onclick/hud/plane_master.dm

```diff
@@ -28,6 +35,10 @@
 	appearance_flags = PLANE_MASTER
 	blend_mode = BLEND_OVERLAY
 
+/obj/screen/plane_master/game_world/Initialize(mapload)
+	. = ..()
+	add_filter("displacer", 1, displacement_map_filter(render_source = GRAVITY_PULSE_RENDER_TARGET, size = 10))
```

Comment:
Isn't this (along with the parent call) going to add two of the same filters on top of one another?

Reply:
Think this one I ment to remove but forgot

Reply:
removed